test/openAPI/features/status_email.feature

```diff
@@ -0,0 +1,135 @@
+@method=GET @endpoint=/status/email
+Feature: Get status of your email
```

Comment:
I would change the feature message on "Get an overview of sent emails statuses" or something similar, the word "your" don't fit here